README.md

```diff
@@ -1,91 +1,87 @@
+    + [What NVIDIA reflex is NOT?](#what-nvidia-reflex-is-not)
+    + [How to enable NVIDIA Reflex for Ghostrunner](#how-to-enable-nvidia-reflex-for-ghostrunner)
+  * [AMD improvements](#amd-improvements)
 - [Resources](#resources)
 - [Useful links](#useful-links)
 
+
 # Frequently asked questions
 
-## How to use the new always available instant restart?
-In settings, ensure instant restart is set to 'on'. If you wish to rebind the key, it's the same entry as for quick restart on death, there is no new entry for this functionality.
+## How do I use the new always available "Instant restart"?
+In settings, ensure the option "Instant restart" is set to "On". If you wish to rebind the key, it's the same entry as for Quick restart on death, there is no new entry for this functionality.
+
+## How do I choose which DirectX version to use?
+By default, Ghostrunner launches in DirectX 11. To choose which DirectX version to use, launch the game via the Steam application and choose the version you want it to launch.
 
-## How can I use directX 12
-To choose which directX to use, launch the game via steam, not a desktop shortcut. Alternatively add "-dx12" (without the quotes) to steam launch options.
+## How to set DirectX 12 by default?
+Open the Steam application, go to Library, right click Ghostrunner, click on Properties, and add `-dx12` in the Steam launch options.
 
-## Should I use 11 or 12
-DX11 for stability. DX12 if you want ray tracing and have an RTX card or dx11 is causing you issues
+## Should I use DirectX 11 or 12?
+DirectX 11 is generally better used for stability. DirectX 12 is better if you want ray tracing, you have an RTX card or DirectX 11 is causing issues.
 
 
 # Technical issues
-## Why are my graphics/keybinds resseting?
-Go to %LOCALAPPDATA%\Ghostrunner\Saved\Config\WindowsNoEditor
-Set the things you want, then right click each file > properties > set as read-only.
+
+## My graphics/keybinds are resetting
+Go to `%LOCALAPPDATA%\Ghostrunner\Saved\Config\WindowsNoEditor`. Set the things you want, then right click each file, go to Properties and set as read-only.
 
 ## Nothing disables Vsync
-This seems to be a rare issue where the in-game setting does not work. No workarounds are known, try to completley wipe the game and config, then reinstall.
+This seems to be a rare issue where the in-game setting does not work. No workarounds are known, try to completely wipe the game and config, then reinstall.
 
-## How to handle the input lag/broken vsync
-Vsync in games tends to be subpar, Nvidia(and AMD for the most part) does it better.
-Also, disable Windows 10 game mode, as well as this: <https://www.windowscentral.com/how-disable-and-remove-game-bar-windows-10-creators-update>
+## Input lag and/or broken VSync
+- Disable VSync. VSync in games tends to be subpar. NVIDIA and AMD have implemented similar technologies to VSync called G-Sync and FreeSync respectively, which are for the most part better. Disable Vsync.
```

Comment:
This is not what I meant initially, Gsync and freesync are a separate thing and they require vsync to be on. What I meant is that you can force Vsync made by AMD/Nvidia via respective options, to use instead of in-game one.